### DISCUSSION TITLE
Pagina CSS Terminada

### DISCUSSION THREAD

**Discussion by IsmaelGomezPerez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#360] [Show and tell]
Me llevo mucho mas tiempo de lo que pense, pero por fin he logrado terminarla. 
Sin lugar a duda que con la practica es cuando realmente se aprende.

[Pagina](https://ismaelgomezperez.github.io/Vaccination-DemoPage/)

El [repositorio](https://github.com/IsmaelGomezPerez/Vaccination-DemoPage) un poco muy revuelto, pero se aprendio sobre la marcha.

**Comment by Dfredude** [comment]
Quedo muy bien !!

**Comment by khisus19** [comment]
Quedó excelente. Y responsiva que fue lo que me faltó a mi. Una recomendación que te doy es que uses imágenes en formato webp y no en png ya que png es un formato que ocupa más datos. Si evalúas tu página con lighthouse verás que tendrá baja calificación por esa razón

**Reply by iRetr0o** [reply]
Puedes meter las imágenes por https://tinypng.com para que tengan el menor peso posible

**Reply by khisus19** [reply]
Aún así webp es el formato más recomendado para el score de lighthouse. Yo hice la prueba usando png pasados por tinypng y webp. Por eso te lo digo.

**Reply by hackit92** [reply]
Es buena opción como alternativa, pero webp presenta algunos problemas de incompatibilidad en navegadores viejos y en safari, por lo que no podríamos descartar del todo png aun

**Reply by IsmaelGomezPerez** [reply]
Muchas gracias por la recomendacion, si tuve algunos problemas con el formato y la transparencia de la imagen, vi que no todos  los formatos lo soportan, para la proxima practica experimentare por ahi

**Comment by lilmendez** [comment]
quedo muy bien, creo que no podre hacer esta practica, me parece muy avanzado para lo que aprendimos en los lives, mas para alguien que esta en cero en todo esto.

**Reply by AngelCruzO** [reply]
Haz lo que puedas hacer, si te rindes no sabrás de lo que eres capaz de realizar

**Reply by IsmaelGomezPerez** [reply]
Eso mismo! hay que intentarlo, yo empece casi desde 0 hace poco mas de tres meses, pero si hay que investigar por nuestra cuenta, porque igual senti que la informacion de los lives es muy superficial, pero dan buenos tips

**Reply by vicogarcia16** [reply]
Claro que tienes que investigar por tu cuenta, pero la exigencia de dicha práctica no iba acorde a lo expuesto. Dicen que es un curso básico de todas las tecnologías Front End pero ya cuando ves las prácticas, mejor sigues de autodidacta. Ojo no digo que la información que exponen no sea buena, digo que debe de haber congruencia porque algunos que estan inscritos a dicho programa no son unos expertos en la materia. Creo que a veces se nos olvida tener un poco de empatía y nos cargamos un ego que para que te cuento.
